R/qenv-eval_code.R

```diff
@@ -37,15 +37,15 @@ setMethod("eval_code", signature = c("qenv", "expression"), function(object, cod
     x <- withCallingHandlers(
       tryCatch(
         {
-          eval(code_line, envir = object@env)
+          eval(parse(text = code_line), envir = object@env)
           NULL
         },
         error = function(e) {
           errorCondition(
             message = sprintf(
               "%s \n when evaluating qenv code:\n%s",
               .ansi_strip(conditionMessage(e)),
-              paste(format_expression(code), collapse = "\n")
+              paste(code, collapse = "\n")
```

Comment:
I think it would be better if we return particular code line instead of the whole `code`. I believe, error message should contain formatted code.

```suggestion
              paste(code_line, collapse = "\n")
```

Reply:
now the code input is length one, so it's gonna be the whole code returned anyway

Reply:
fb0dd19605250ac37886fc10324a1a2cc61e5aca